ksqldb-rest-app/src/main/java/io/confluent/ksql/api/impl/BlockingQueryPublisher.java

```diff
@@ -78,9 +76,23 @@ public void setQueryHandle(final QueryHandle queryHandle, final boolean isPullQu
     this.queryId = queryHandle.getQueryId();
     this.queue.setQueuedCallback(this::maybeSend);
     this.queue.setLimitHandler(() -> {
+      if (isPullQuery) {
+        if (queryHandle.getConsistencyOffsetVector().isPresent()) {
+          log.info("Limit handler: Add consistency token to queue");
```

Comment:
These logs will only happen if consistency is enabled and currently nobody will enable it since it is not available yet. Moreover, these are there only to debug the problem. Once it doesn't happen anymore, I will remove most of them